Question 1.txt

```diff
@@ -0,0 +1,10 @@
+//This is question 1, it's all about theory! #F**kIt!
+
+What's the different between COMBINATORIAL and SEQUENTIAL circuit.
+
+Answer in my form. The real answer will show in this text file (and yes, you can download all of this solution that
+I provided in this GitHub everytime!)
+
+Good Luck Guys!
+
+Answer: 
```

Comment:
Answer: The combinatorial circuit is the circuit that doesn't use the clock or flipflops gate. It uses mostly in analogue and digital system.